spec/components/admin/schools/cohort_component_spec.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+RSpec.describe Admin::Schools::CohortComponent, type: :component do
+  let(:school) { FactoryBot.create(:seed_school) }
+  let(:cohort) { FactoryBot.create(:seed_cohort) }
+  let(:school_cohort) { FactoryBot.create(:seed_school_cohort, school:, cohort:) }
+  let(:partnership) { FactoryBot.create(:seed_partnership, :valid, school:, cohort:) }
+  let(:relationships) { FactoryBot.create_list(:seed_partnership, 2, :valid, school:, cohort:, relationship: true) }
+  let(:partnerships_and_relationships) { [partnership, *relationships] }
+  let(:kwargs) { { school:, cohort:, school_cohort:, partnerships_and_relationships: } }
+
+  let(:relationship_matcher) { ".relationship" }
+  let(:partnership_matcher) { ".partnership" }
```

Comment:
Is there a way to write the tests whilst avoiding needing these extra classes in the HTML?